_posts/2020-09-14-release-0-21-4.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "JuMP 0.21.4 is released"
+date:   2020-09-14
+categories: [releases]
+permalink: /blog/0.21.4-release/
+---
+
+We are happy to announce the joint release of JuMP 0.21.4 and MathOptInterface
+0.9.15! These releases are a mix of new features and some much needed
```

Comment:
Perhaps mention that this release closes a number of issues on the roadmap to JuMP 1.0.